File: WebsiteAppRPG/WebsiteAppRPG.Frontend/website-app-rpg-frontend/src/models/player.ts
export class PlayerColors{
    
    public playerRedColor: Number;
    
    public playerGreenColor: Number;
    
    public playerBlueColor: Number;

    public playerAlphaLevel: Number;

    constructor(redColor: Number, greenColor: Number, blueColor: Number, alphaLevel: Number) {
        this.playerRedColor = redColor;
        this.playerGreenColor = greenColor;
        this.playerBlueColor = blueColor;

        this.playerAlphaLevel = alphaLevel;
    }
}

export class Player{

    public playerID: number;

    public email: string;

    public name: string;

    public password: string;

    public positionX: number;

    public positionY: number;

    constructor(playerID: number, email: string, name: string, password: string, positionX: number, positionY: number)
    {
        this.playerID = playerID;
        this.email = email;
        this.name = name;
        this.password = password;
        this.positionX = positionX;
        this.positionY = positionY;
    }
}

File: WebsiteAppRPG/WebsiteAppRPG.Core/Entities/Player.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WebsiteAppRPG.WebsiteAppRPG.Core.Entities
{
    [Table("Players")]
    public class Player
    {
        [Key]
        [Column("PlayerID")]
        public int PlayerID { get; set; }

        [Column("Email")]
        public string Email { get; set; }

        [Column("PlayerName")]
        public string Name { get; set; }

        [Column("Password")]
        public string Password { get; set; }

        [Column("PositionX")]
        public int PositionX { get; set; }

        [Column("PositionY")]
        public int PositionY { get; set; }

        public Player(string email, string name, string password, int positionX, int positionY)
        {
            Email = email;
            Name = name;
            Password = password;
            PositionX = positionX;
            PositionY = positionY;
        }
    }
}
